config/resolve_credentials_test.go

```diff
@@ -87,6 +101,44 @@ func setupCredentialsEndpoints(t *testing.T) (aws.EndpointResolver, func()) {
 	}
 }
 
+func ssoTestSetup() (func(), error) {
+	dir, err := ioutil.TempDir("", "sso-test")
+	if err != nil {
+		return nil, err
+	}
+
+	cacheDir := filepath.Join(dir, ".aws", "sso", "cache")
+	err = os.MkdirAll(cacheDir, 0750)
+	if err != nil {
+		os.RemoveAll(dir)
+		return nil, err
+	}
+
+	tokenFile, err := os.Create(filepath.Join(cacheDir, "eb5e43e71ce87dd92ec58903d76debd8ee42aefd.json"))
```

Comment:
qq: Should `eb5e43e71ce87dd92ec58903d76debd8ee42aefd` be a randomized string instead?

Reply:
This needs to match the startURL content of the profile, in this case this is that sha1 of `https://127.0.0.1/start`